core/workspace_svg.js

```diff
@@ -1686,28 +1688,34 @@ Blockly.WorkspaceSvg.setTopLevelWorkspaceMetrics_ = function(xyRatio) {
 };
 
 /**
- * Update whether this workspace has resizes enabled.
- * If enabled, workspace will resize when appropriate.
- * If disabled, workspace will not resize until re-enabled.
- * Use to avoid resizing during a batch operation, for performance.
- * @param {boolean} enabled Whether resizes should be enabled.
+ * Set whether this workspace is currently updating it's contents.
+ * If updating, workspace will not resize until no longer updating.
+ * If not updating, workspace will resize when appropriate.
+ * Use to avoid resizing and other logic during a batch operation, for
+ * performance.
+ * @param {boolean} isUpdating Whether the workspace is updating.
  */
-Blockly.WorkspaceSvg.prototype.setResizesEnabled = function(enabled) {
-  var reenabled = (!this.resizesEnabled_ && enabled);
-  this.resizesEnabled_ = enabled;
-  if (reenabled) {
-    // Newly enabled.  Trigger a resize.
+Blockly.WorkspaceSvg.prototype.setIsUpdating = function(isUpdating) {
```

Comment:
I'm good with start/endBulkUpdate. I'll get back to this another day when I have some time.